src/fast_check/transform.rs

```diff
@@ -144,7 +140,7 @@ pub fn transform(
     parsed_source.text_info().text_str().to_owned(),
   );
   let program = Program::Module(module);
-  let EmittedSource { text, source_map } = emit(
```

Comment:
I removed this because I got confused about what `source_map` was referring to earlier (because above it's a different kind of source map)